[Docs][Kuberay] Documentation for Using Kuberay with TPUs (#45924)
ray-project/ray: doc/source/cluster/kubernetes/user-guides/k8s-cluster-setup.md

aws-eks-gpu-cluster
gcp-gke-gpu-cluster
gcp-gke-tpu-cluster
```

> Most KubeRay documentation only requires a local Kubernetes cluster such as [Kind](https://kind.sigs.k8s.io/).

Reviewer: ```suggestion
Most KubeRay documentation examples only require a local Kubernetes cluster such as [Kind](https://kind.sigs.k8s.io/).
```

> # Setting up a GKE cluster (Google Cloud)

Reviewer: ```suggestion
# Set up a GKE cluster (Google Cloud)
```

- {ref}`kuberay-gke-gpu-cluster-setup`
- {ref}`kuberay-gke-tpu-cluster-setup`

(eks-setup)=

> # Setting up an EKS cluster (AWS)

Reviewer: ```suggestion
# Set up an EKS cluster (AWS)
```
